jq.py

```diff
@@ -149,12 +152,17 @@ def outputDataAsString(debugger, symbol):
     interpreter = debugger.GetCommandInterpreter()
     commandOutput = lldb.SBCommandReturnObject()
     
-    commandString = "po String(data: %s, encoding: String.Encoding.utf8)!" % (symbol)
+    commandString = "po String(data: %s, encoding: String.Encoding.utf8)" % (symbol)
```

Comment:
A faster way to check for nil is if commandString is `"po (%s == nil)" % (symbol)`

Then check for `true` or `false`. Don't need to create a String, and then discard it.